DataStructures/HashMap/Hashing/LinkedHashMap.java

```diff
@@ -0,0 +1,35 @@
+    import java.util.*;    
```

Comment:
@suranasourabh Kindly provide a description and a sample case for your repository